tests/migrations/test_migrations_conflict_long_name/0002_second2.py

```diff
@@ -0,0 +1,17 @@
+from django.db import migrations, models
```

Comment:
We could use a long names instead of adding many migrations, e.g.
```
- 0001_initial.py
- 0002_second_create_model_something.py
- 0002_conflicting_second_migration_with_very_long_name.py
```